js/classes/Calendar.js

```diff
@@ -1095,13 +1095,22 @@ class Calendar
             if (hasLunchStarted) return true;
             dayValues.push(dayEnd);
         }
+        
+        
+        // Adding fix to implement test pass
+        // Means no valid time was entered
```

Comment:
I feel like this comment is more like something that would belong on the description of the PR, not really the code. In the code we don't really want to change behavior to make a test work, but the other way around, in which the test will test a behavior.
I think a comment like the following should fit better:

```suggestion
        // If non-empty inputs are passed, they should be valid (be on the day values)
```